parser/parser.go

```diff
@@ -32,6 +32,17 @@ var parserFactory = map[string]func() Parser{
 	"gitlabci":   func() Parser { return new(GitLabCI) },
 }
 
+// GetAvailableParser return the sorted list of available parser.
+func GetAvailableParsers() []string {
```

Comment:
This function already exists - see `List`. It's identical :)

Reply:
Sorry, my fault